common/app/common/configuration.scala

```diff
@@ -136,17 +144,7 @@ class GuardianConfiguration extends Logging {
       configuration.getIntegerProperty("tag_indexes.rebuild_rate_in_minutes").getOrElse(60)
   }
 
-  object environment {
-    private val installVars = new File("/etc/gu/install_vars") match {
-      case f if f.exists => IOUtils.toString(new FileInputStream(f))
-      case _ => ""
-    }
-
-    private val properties = Properties(installVars)
-
-    def apply(key: String, default: String) = properties.getOrElse(key, default).toLowerCase
-
-    val stage = apply("STAGE", "unknown")
+  object environment extends InstallVars.InstallationVars {
```

Comment:
well, so far it's true that we only need one, so we don't need to worry about where the others belong until someone needs them.  We can revisit at that point.  It's a balance between bringing them in explicitly, or dragging them all in with a trait.  Or, to avoid the problem entirely, we can alter the 23 places to use installvars directly and remove from environment.